perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -59,6 +59,30 @@ sub new_report_title_field_hint {
 
 sub report_sent_confirmation_email { 'external_id' }
 
+=item * We don't send report sent emails for Jadu backed contacts.
+
+=cut
+
+sub suppress_report_sent_email {
```

Comment:
Is this new method needed? Can't you do the same check in the existing `report_sent_confirmation_email` method and return the appropriate value there?

Reply:
If `report_sent_confirmation_email` returns a falsey value, a report logged email is sent straight after a report is confirmed (`New.pm`). If truthy, a report logged email is sent after the report is sent (`Queue/Item/Report.pm`) instead, using the given string to extract an ID to include in the email.

Beds want no report logged email sent at all, hence the new method.

Reply:
Ah, shame we can't have a trinary data type :) Makes sense! 👍